config/environments/production.rb

```diff
@@ -66,6 +66,15 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { :host => 'tutoring-session-staging.herokuapp.com' }
+
+config.action_mailer.delivery_method = :smtp
+config.action_mailer.smtp_settings = {
+  :address => "127.0.0.1",
+  :port    => 25,
+  :domain  => 'yourdomain.com'
+}
```

Comment:
Does this not need to be edited to actually be a proper address and domain, since these are the settings for production?

Reply:
These need to be changed for production

Reply:
Alright then, I will wait to merge this PR until the configs are set up properly.

Reply:
is the domain different between the staging and production areas?

Reply:
You can check Heroku, but I'm fairly confident they are.